app/services/users/invitation_service.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Users
+  class InvitationService < ApplicationService
+    def call
+      invite = User.invite!(invite_params) { |u| u.skip_invitation = true }
+      if invite.errors.empty?
+        invite.create_family(family_params.merge(owner: current_user.id))
+        invite.update_attribute(:family_id, invite.family.id)
+        invite.update_attribute(:invited_by_id, current_user.id)
```

Comment:
`update_attributes`

Reply:
i need to skip validation when i create invited user, so that is why i used update_attribute but not update_attributes

Reply:
ok, then use `update_columns`

Reply:
done